doc/holochain_101/book.toml

```diff
@@ -3,3 +3,6 @@ authors = ["David Meister"]
 multilingual = false
 src = "src"
 title = "holochain_101"
+
+[build]
+preprocess = ["links"]
```

Comment:
what does this do?

Reply:
@Connoropolous It adds the prev/next page links to the MDBOOK.

Reply:
Cool, I like it